tasks/receptionist/src/receptionist/state_machine.py

```diff
@@ -39,7 +39,7 @@ def __init__(
             }
             self.userdata.guest_name = "zoe"
             self.userdata.dataset = "receptionist"
-            self.userdata.confidence = 0.15
+            self.userdata.confidence = 0.2
```

Comment:
Would be nice if we made these a parameter of the constructor, or loaded through a config.